codespell_lib/__main__.py

```diff
@@ -5,5 +5,7 @@
 if __name__ == "__main__":
     try:
         sys.exit(_script_main())
+    except BrokenPipeError:
+        pass
```

Comment:
Perhaps just:
```python
    except (KeyboardInterrupt, BrokenPipeError):
        pass
```

Reply:
yes, that's better I'm not that good a Python

Reply:
done